### DISCUSSION TITLE
How to delete LazyVim from NeoVim?

### DISCUSSION THREAD

**Discussion by Arijit1st** [neovim/neovim#31388] [Q&A]
I am new to NeoVim. I used the LazyVim distribution to get a taste of It, but now I understand how it works, and I want to configure NeoVim all by myself. However, I cannot figure out how to delete LazyVim.

I even uninstalled NeoVim and installed it again, but still when I type 'nvim' LazyVim automatically starts. Can you please tell me how to delete LazyVim?

**Comment by dundargoc** [comment]
You should ask them: https://github.com/LazyVim/LazyVim/discussions

Quick google search: https://www.reddit.com/r/neovim/comments/1baoyjt/how_can_uninstall_lazyvim/?rdt=43605

**Reply by Arijit1st** [reply]
Thanks for your prompt comment! In which directory of NeoVim should we execute those commands that were there in the Reddit answer?

**Reply by dundargoc** [reply]
It works in any directory, it's an absolute path

**Reply by Arijit1st** [reply]
> It works in any directory, it's an absolute path

I am on Windows, so the commands are different, and the path does not work.
